web/packages/demo/www/index.html

```diff
@@ -25,7 +25,7 @@
             }
 
             #main {
-                margin: 15px;
+                margin-top: 15px;
```

Comment:
What is the purpose of this change?

Reply:
Currently the demo page doesn't fit in a whole page (for me at least), so it has a very annoying scrollbar.
This change eliminates the scrollbar, but in second thought, maybe it would be better to define `margin-bottom` on `#nav` instead.